doc/user/content/self-hosted/install-on-local-kind.md

```diff
@@ -0,0 +1,246 @@
+
+1. Start Docker if it is not already running.
+
+1. Open a Terminal window.
+
+1. Create a kind cluster.
+
+   1. Create a configuration file named `kind-config.yaml` with the following
+      content:
+
+      ```yaml
+      kind: Cluster
+      apiVersion: kind.x-k8s.io/v1alpha4
+
+      name: my-local-kind
+
+      nodes:
+        - role: control-plane
+          kubeadmConfigPatches:
+            - |
+              kind: InitConfiguration
+              nodeRegistration:
+                  kubeletExtraArgs:
+                    node-labels: "materialize.cloud/disk=true,workload=materialize-instance"
+      ```
+
+    1. Create the cluster using the configuration file.
+
+      ```shell
+      kind create cluster --config kind-config.yaml
+      ```
+
+1. Create the `materialize` namespace.
+
+   ```shell
+   kubectl create namespace materialize
+   ```
+
+1. Install the Materialize Helm chart using the files provided in the
+   Materialize repo.
+
+   1. Go to the Materialize repo directory.
+
+   1. *Temporary*. For MacOS, edit the `misc/helm-charts/operator/values.yaml`
+      file to set `networkPolicies.enabled` to `false`.
+
+      ```yaml
+      # Network policies configuration
+      networkPolicies:
+       # Whether to enable network policies for securing communication between pods
+       enabled: false
+      ```
+
+   1. Install the Materialize operator with the release name
+      `my-materialize-operator`:
+
+      ```shell
+      helm install my-materialize-operator -f misc/helm-charts/operator/values.yaml misc/helm-charts/operator
+      ```
+
+   1. Verify the installation and check the status:
+
+      ```shell
+      kubectl get all -n materialize
+      ```
+
+      Wait for the components to be in the `Running` state:
+
+      ```shell
+      NAME                                           READY   STATUS              RESTARTS   AGE
+      pod/my-materialize-operator-776b98455b-w9kkl   0/1     ContainerCreating   0          6s
+
+      NAME                                      READY   UP-TO-DATE   AVAILABLE   AGE
+      deployment.apps/my-materialize-operator   0/1     1            0           6s
+
+      NAME                                                 DESIRED   CURRENT   READY   AGE
+      replicaset.apps/my-materialize-operator-776b98455b   1         1         0       6s
+      ```
+
+      If you run into an error during deployment, refer to the
+      [Troubleshooting](/self-hosted/troubleshooting) guide.
+
+1. Install PostgreSQL and minIO.
+
+    1. Go to the Materialize repo directory.
+
+    1. Use the provided `postgres.yaml` file to install PostgreSQL as the
+       metadata database:
+
+        ```shell
+        kubectl apply -f misc/helm-charts/testing/postgres.yaml   
+        ```
+
+    1. Use the provided `minio.yaml` file to install minIO as the blob storage:
+
+        ```shell
+        kubectl apply -f misc/helm-charts/testing/minio.yaml   
+        ```
+
+1. Install the following metrics service:
```

Comment:
I realize that the [testing/readme.md](https://github.com/MaterializeInc/materialize/blob/main/misc/helm-charts/testing/README.md#metrics-service) says we need this, but this errors for me on my mac.

Reply:
metrics-server should be optional... it's needed for some system table metrics to work, but not blocking to testing materialize